integration-test/src/test/java/org/jboss/pnc/integration/WebSocketsNotificationTest.java

```diff
@@ -90,9 +90,9 @@ public void before() throws Exception {
     @Test
     public void shouldConnectToWebSockets() throws Exception {
         //given
-        BuildStatusChangedEvent buildStatusChangedEvent = new DefaultBuildStatusChangedEvent(BuildStatus.NEW, BuildStatus.BUILD_WAITING, 1, null);
+        BuildStatusChangedEvent buildStatusChangedEvent = new DefaultBuildStatusChangedEvent(BuildStatus.NEW, BuildStatus.BUILD_COMPLETED_SUCCESS, 1, 1);
         // How to mock BuildRecordProvider?
-        String expectedJsonResponse = "{}";//"{\"oldStatus\":\"NEW\",\"newStatus\":\"BUILD_WAITING\",\"buildTaskId\":1,\"buildExecution\":null}";
+        String expectedJsonResponse = "{\"payload\":{\"id\":1,\"eventType\":\"BUILD_COMPLETED\",\"userId\":1}}";
```

Comment:
@alexcreasy Please take a look at final version of a build status notification.. Is it ok?